src/components/Featured/Featured.tsx

```diff
@@ -0,0 +1,65 @@
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import { FeaturedFragment$key } from '~/generated/FeaturedFragment.graphql';
+
+import breakpoints from '../core/breakpoints';
+import { VStack } from '../core/Spacer/Stack';
+import FeaturedSection from './FeaturedSection';
+
+type Props = {
+  queryRef: FeaturedFragment$key;
+};
+
+export default function Featured({ queryRef }: Props) {
+  const query = useFragment(
+    graphql`
+      fragment FeaturedFragment on Query {
+        trendingUsers7Days: trendingUsers(input: { report: LAST_7_DAYS }) {
+          ... on TrendingUsersPayload {
+            __typename
+            # eslint-disable-next-line relay/must-colocate-fragment-spreads
+            ...FeaturedListFragment
+          }
+        }
+        trendingUsersAllTime: trendingUsers(input: { report: ALL_TIME }) {
+          ... on TrendingUsersPayload {
+            __typename
+            # eslint-disable-next-line relay/must-colocate-fragment-spreads
+            ...FeaturedListFragment
+          }
+        }
+        # eslint-disable-next-line relay/must-colocate-fragment-spreads
```

Comment:
@tbezman 
I wasn't 100% sure if should be disabling these errors.

I'm querying for the 2 reports using the trendingUsers query and displaying each using the `FeaturedSection` component. Im using  `FeaturedSection` as a presentational component so Im not declaring the fragments there, but rather in the child components like `FeaturedUserCard`. seems like that creates too much distance between the spread and the fragment declaration, but moving it a level or two down didn't resolve the issue for me so I just kept it here. 
Lmk if there's a better way to handle this

Reply:
A fragment should spread its direct children only. Skipping levels is not sound by the lint rules. 

It just follows the rules of "A component should only know about its children, not its children's children"

I added the fragments for you. If this feels like a lot of friction, I recommend creating some snippets in VSCode to make defining fragments really fast. 

Here's mine in IntelliJ

```
const $LOWER_TYPE$ = useFragment(graphql`
    fragment $FileName$$FragmentName$Fragment on $TYPE$ {
    }
`, $LOWER_TYPE$Ref)
```

Reply:
I just type `uf` and press tab and it does a lot of work for me. Auto import does the rest of the work.

I also have one for defining a new component 

```
import { useFragment } from 'react-relay';
import { graphql } from 'relay-runtime';
import { $FileName$$FragmentName$Fragment$key } from '~/generated/$FileName$$FragmentName$Fragment.graphql';


type $Component$Props = {
    $LOWER_TYPE$Ref: $FileName$$FragmentName$Fragment$key;
};

export function $Component$({ $LOWER_TYPE$Ref }: $Component$Props) {
    const $LOWER_TYPE$ = useFragment(graphql`
        fragment $FileName$$FragmentName$Fragment on $TYPE$ {
            __typename
        }
    `, $LOWER_TYPE$Ref)

    return null;
}
```